8.x/theme-development/products.md

```diff
@@ -1,3 +1,435 @@
+**10**| `invoices`             |Invoice[]                         |Specifies what factors the product is used for.
+**11**| `wishLists`            |CustomerUser[]                    |The list that the user wants to create in the future.
+**12**| `needLists`            |CustomerUser[]                    |List that the user wants to provide the product but is not available.
+**13**| `cartRows`             |CartRow[]                         |Specifies in which basket the product is found.
+**14**| `tags`                 |Tag[]                             |Product tags.
+**15**| `badges`               |Badge[]                           |Badges that are displayed on the product.
+**16**| `review`               |Review                            |Product reviews that include text from user reviews as well as the rates they have given to the product.
+**17**| `customerMetaCategor`  |CustomerMetaCategory              |Information the customer needs to provide to buy a particular product.
+**18**| `discountGroup`        |DiscountGroup                     |Plan of reducing the prices of the products.
+**19**| `priority`             |integer                           |Indicates the priority of the product stored within the system.
+**20**| `productPackage`       |ProductPackage                    |:question:
+**21**| `productPackages`      |ProductPackage[]                  |:question:
+
+### Display product properties
+
+The complete directory structure of the template for this project is shown below. The template directory has two subdirectories, **public**, and **resources**, which are the Boilerplate of this project. For more information on creating a layout and its steps, see [5 minute quick start](https://docs.larammerce.com/8.x/theme-development/).
+
+```bash
+|---node_modules/
+|---public/
+     |---HCMS-assets/
+     |---views/
+|---resources/
+     |---assets/
+          |---css
+          |---js
+          |---sass
+|---.envrc
+|---.envrc.example
+|---.gitignore
+|---composer.json
+|---deploy.sh
+|---mix-manifest.json
+|---package.json
+|---package-lock.json
+|---webpack.mix.js
+```
+
+**TIP:** *Boilerplate is any text, documentation, or procedures that can be reused more than once in a new context without any substantial changes to the original.* *<sup>[1](#1)</sup>* 
+
+**TIP:** *The blade is the simple, yet powerful templating engine that is included with Laravel.*  *<sup>[2](#2)</sup>* 
+
+Presently if you enter the product's single page, **how can you show the item properties?**
+Larammerce will tell you.
+
+if you click on a product in the store, you will enter a single page with a dedicated URL that displays content from the blade called `product-single.blade.php`. The procedure, in general, is as which if you enter the `product-single.blade.php` file in the `public/views` path of the Boilerplate template, you can display the properties on the single product page by calling the properties.
```

Comment:
There is no need to use the word boilerplate all over the document. I mean, you can just talk about the template. Refer to the template boilerplate once at the starting point of the paper. So in other parts, there should be a replacement.